fails on ppc64le:
ERROR Unresolved dependencies.
//third_party/boringssl:test_support(//build/toolchain/linux/unbundle:default)
  needs //third_party/boringssl:test_support_asm(//build/toolchain/linux/unbundle:default)

and we don't run the tests
--
--- a/third_party/boringssl/BUILD.gn
+++ b/third_party/boringssl/BUILD.gn
@@ -196,7 +196,6 @@
 
     deps = [
       ":boringssl",
-      ":test_support_asm",
       "//base",
       "//testing/gtest",
     ]
